sdp.c

```diff
@@ -654,7 +656,7 @@ int janus_sdp_anonymize(janus_sdp *anon) {
 		GList *purged_ptypes = NULL;
 		while(tempA) {
 			janus_sdp_attribute *a = (janus_sdp_attribute *)tempA->data;
-			if(strstr(a->value, "red/90000") || strstr(a->value, "ulpfec/90000") || strstr(a->value, "rtx/90000")) {
+			if(a->value && (strstr(a->value, "red/90000") || strstr(a->value, "ulpfec/90000") || strstr(a->value, "rtx/90000"))) {
```

Comment:
I had not incorporated this change of yours from another PR as in my SDP utils you can't have a line like `a=name:`, that is, without a value after the semicolon: the parser returns an error. Anyway, now that I think about it this iterator may actually also traverse regular attributes with just a name (e.g., `a=sendrecv`), which means that check is indeed needed :+1: